di-ipv-orchestrator-stub/src/main/java/uk/gov/di/ipv/stub/orc/utils/JWTSigner.java

```diff
@@ -0,0 +1,102 @@
+package uk.gov.di.ipv.stub.orc.utils;
+
+import com.nimbusds.jose.*;
```

Comment:
Also can we remove the star import here. I think this is just a setting in intellij that you need to change otherwise it will keep auto importing the whole package when you use over like 3 or 4 dependencies